server/controllers/appearsInController.js

```diff
@@ -0,0 +1,66 @@
+var AppearsIn = require('../models/AppearsIn');
+
+exports.index = function(req, res) {
+    AppearsIn.find({})
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((result) => {
+        res.status(200).json(result);
+    });
+};
+
+exports.show = function(req, res) {
+    AppearsIn.findById(req.params.appearsin_id)
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((result) => {
+        res.status(200).json(result);
+    });
+};
+
+exports.create = function(req, res) {
+    var appearsin = new AppearsIn(req.body);
+    appearsin.save()
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((createdAppearsIn) => {
+        res_json = {
+            "message": "AppearsIn created",
+            "data": {
+                "appearsInId": createdAppearsIn._id,
+            }            
+        }
+        res.status(200).send(res_json);
```

Comment:
Se for enviar json na response, acredito que usar res.status(xxx).**json**() seja a convenção (em vez de .send()).